src/test/java/baseball/ApplicationTest.java

```diff
@@ -15,7 +15,7 @@ class ApplicationTest extends NsTest {
         assertRandomNumberInRangeTest(
                 () -> {
                     run("246", "135", "1", "597", "589", "2");
-                    assertThat(output()).contains("낫싱", "3스트라이크", "1볼 1스트라이크", "3스트라이크", "게임 종료");
+                    assertThat(output()).contains("nothing", "3strike", "1ball 1strike", "3strike", "game over");
```

Comment:
oh this is necessary. good job

Reply:
'good job' is rude to someone who is like a supervisor